docs/Contributing.adoc

```diff
@@ -6,11 +6,38 @@ Herein lies the contribution guidelines for helping out with this project. Do ta
 
 === General rules
 
-* The Ansible <<coc,Code of Conduct>> still applies.
-* For git messages, branch names, ..., follow <<gitstyle,Git Style Guide>>
-* Should you wish to work on a completely new standard, GREAT, but please file an issue for tracking.
+* The Ansible link:https://docs.ansible.com/ansible/latest/community/code_of_conduct.html[Code of Conduct] still applies.
+* For git messages, branch names, ..., follow link:https://github.com/agis/git-style-guide[Git Style Guide].
+* Should you wish to work on a completely new standard, GREAT, but please file an **issue** for tracking. In your Pull Request, specify `closes #NUM` to automatically close the issue when the PR is merged.
 * To contribute, fork and make a pull request against the `development` branch.
-* All tasks should be in YAML literal.
+*  Use link:https://asciidoctor.org/docs/asciidoc-writers-guide/[asciidoc] for documentation.
+* Pull requests should only change one role, one config, or one role as applied to one config. There may be exceptions to this rule, but you must explain why a special exception should apply to a PR. Ask yourself, « Can this pull request be separeted in several smaller pull requests ? »
+* Pull Request titles:
+** Must start with `Add | Change | Fix | HOTFIX | Remove`
+** followed by `config | workload | role | core | test | documentation`
+** followed by the **name** of the config, workload, cloud provider, test or documentation
+** followed by a short description
+* Pull Request must be tested. Explain how you tested your change.
+* Owner of role or config must be identified.
+* If your Pull Request is not ready for review, open it as link:https://github.blog/2019-02-14-introducing-draft-pull-requests/[Draft]] or add `WIP` in the title.
+
+=== Code Quality Rules
+
+. a YAML `.yamllint` file should be added to every role and config when any substantial change is applied to the role or config, all new roles and configs must include a `.yamllint`.
+. All tasks should be in YAML literal.
```

Comment:
This means no commandline Ansible format, right? No `foo=bar` notation? Not sure how to state that, but need to clarify I think.

Reply:
done